### DISCUSSION TITLE
is playground v2 supported yet?

### DISCUSSION THREAD

**Discussion by yesbroc** [lllyasviel/Fooocus#1284] [Q&A] [answered]
for better context: [https://huggingface.co/playgroundai/playground-v2-1024px-aesthetic](https://huggingface.co/playgroundai/playground-v2-1024px-aesthetic)

**Comment by MonzonMedia** [comment, marked as answer]
It's just like any other base model, install it into your checkpoints folder. There are a couple fine tuned models on civitai that uses PGv2 as the base. Works pretty well!
